packages/@sanity/default-login/src/LoginWrapper.js

```diff
@@ -79,7 +81,7 @@ export default class LoginWrapper extends React.PureComponent {
           <LoginDialog
             title={this.props.title}
             description={this.props.description}
-            sanityLogo={sanityLogo}
```

Comment:
Shouldn't this one be:
```
SanityLogo={SanityLogo}
sanityLogo={sanityLogo}
```

Reply:
In loginwrapper we have defaultprop on SanityLogo (SanityStudioLogo), so if you send in sanityLogo it should override.